single-build-installer-collect.bat

```diff
@@ -208,6 +208,10 @@ echo "* copy Qt OpenSSL backend plugin"
 start "copy qopensslbackend.dll" /D "%MY_COLLECT_PATH%/" /B /wait cp -af "%CRAFT_PATH%/plugins/tls/qopensslbackend.dll" "%MY_COLLECT_PATH%/tls"
 if %ERRORLEVEL% neq 0 goto
 
+echo "* copy b2-1.dll"
```

Comment:
where is that coming from?

Reply:
this is a new dependency
I did not check how it became mandatory but this is an indirect dependency